source/reference/api/resume.txt

```diff
@@ -71,12 +71,12 @@ Response
 Behavior
 --------
 
+- When you send a ``resume`` request, ``mongosync`` might take at least two 
+  minutes before resuming the migration.
```

Comment:
[Question] 

Would it be reasonable to mention here that users can use the `/progress` endpoint to see if the migration has resumed?

Reply:
Definitely! Will update with that information.